Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -357,7 +358,7 @@ void AODProducerWorkflowDPL::fillTrackTablesPerCollision(int collisionID,
 
 void AODProducerWorkflowDPL::fillIndexTablesPerCollision(const o2::dataformats::VtxTrackRef& trackRef, const gsl::span<const GIndex>& GIndices)
 {
-  for (int src : {GIndex::Source::MFTMCH, GIndex::Source::MCH, GIndex::Source::MFT}) {
+  for (int src : {GIndex::Source::MFTMCH, GIndex::Source::MCH, GIndex::Source::MFT, GIndex::Source::MCHMID}) {
```

Comment:
If I am not wrong, we must loop over sources in the same order as in fillTrackTablesPerCollision, so MCHMID should be first.